recipe/meta.yaml

```diff
@@ -1,45 +1,45 @@
 {% set name = "whatrecord" %}
-{% set version = "0.4.4" %}
+{% set version = "0.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 60dd887f45e282000b3f7b65b877677532bef237c35442b28dc3306b4292aee5
+  sha256: 6c2ff2b9993162a80a07f2c599ce824b979396a6a4194dd0b8b95449e8da4a5f
 
 build:
   entry_points:
-  - whatrecord = whatrecord.bin.main:main
+    - whatrecord = whatrecord.bin.main:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 0
 
 requirements:
   host:
-  - python >=3.9
-  - pip
-  - setuptools >=45
-  - setuptools_scm >=6.2
+    - python >=3.9
+    - pip
+    - setuptools >=45
+    - setuptools_scm >=6.2
   run:
-  - python >=3.9
-  - aiohttp
-  - apischema
-  - epicsmacrolib
-  - graphviz
-  - jinja2
-  - lark
-  - python-graphviz <0.18
+    - python >=3.9
+    - aiohttp
+    - apischema
+    - epicsmacrolib
+    - graphviz
+    - jinja2
+    - lark
```

Comment:
```suggestion
    - lark-parser
```

Reply:
Maybe we should wait for Ken's input before adjusting/merging this one

Reply:
`lark` is actually the correct one; whatrecord has it wrong:
https://anaconda.org/conda-forge/lark-parser/files
vs
https://anaconda.org/conda-forge/lark/files
I think they were later able to claim the name and left the old one to rot